### PR TITLE
chore(ckbtc): add Bitcoin Checker canister

### PR DESCRIPTION
**Pull request by ninegua**
Add Bitcoin Checker canister to both mainnet and testnet configurations with newly created canister ids.

Add support for Bitcoin Checker canister to proposal-cli, and remove KYT canister from it.

### CODE REVIEW

**Comment by ninegua** [comment]
Tested the proposal-cli manually, and confirm that the argument it produces can be used to install wasm to the testnet canister for Bitcoin Checker.

File: rs/cross-chain/proposal-cli/BUILD.bazel [hunk 1]
```diff
@@ -46,7 +46,7 @@ rust_test(
     name = "unit_tests",
     crate = ":make_proposal",
     data = [
-        "//rs/bitcoin/ckbtc/kyt:kyt.did",
```
Comment (gregorydemay) [inline rs/cross-chain/proposal-cli/BUILD.bazel:49]: Do we maybe still want to keep the old KYT around to be able in a few weeks/months to easily retire the old KYT canister, which would require an NNS proposal?

**Review by gregorydemay — APPROVED** [review]

File: rs/cross-chain/proposal-cli/BUILD.bazel (hunk 1 above)
Comment (ninegua) [reply rs/cross-chain/proposal-cli/BUILD.bazel:49]: Does proposal-cli offer uninstall? I think if it is infrequent enough, maybe just manually write a proposal and use ic-admin

Comment (gregorydemay) [reply rs/cross-chain/proposal-cli/BUILD.bazel:49]: > Does proposal-cli offer uninstall?

Good point it doesn't. I agree it's a one-off thing, so not worth it to integrate to proposal-cli.

**Review by THLO — APPROVED** [review]

**Pull request merged**